### DISCUSSION TITLE
0.28 Markdown显示代码异常

### DISCUSSION THREAD

**Discussion by foryouos** [theme-shoka-x/hexo-theme-shokaX#117] [Q&A] [answered]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/58133210/d5a7d70d-3bc0-49ed-b920-521162f61077)
顶部一行显示问题，好像是左上角的点还有语言的问题，不知道该怎么改，重装了Markdown，Markdown语句检查了没有问题，不知道怎么回事嘞？

**Comment by D-Sketon** [comment, marked as answer]
请问_config.yml中highlight和prismjs的都设置为false了吗
如果是hexo s 命令查看的话，建议在执行之前运行一下hexo cl 清除缓存

**Reply by foryouos** [reply]
谢谢啦！应该就是忘记clean清除缓存的原因
